woocommerce/woocommerce-gutenberg-products-block
# Fix visual border for products image placeholder

This PR fixes the issue opened in https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/3354 by adding a border to the placeholder image of "1px solid #f2f2f2" style to match the WooCommerce original theme display. 

The border is applied to placeholder images identified by having an empty `alt` text.

This PR also adds a fallback alternative text as product name value for images missing that value.



Fixes #3354




#### Accessibility




### Screenshots



Blocks that have been tested so far:

## All Products
<img width="1498" alt="Screenshot 2021-10-15 at 11 06 06" src="https://user-images.githubusercontent.com/537751/137453810-a9127729-17b7-48f4-b621-282c0d527af1.png">

## Products By Category
<img width="1149" alt="Screenshot 2021-10-15 at 11 06 40" src="https://user-images.githubusercontent.com/537751/137453848-46cab4ef-4127-48b1-98cc-69b3571c422d.png">

## Newest Products
<img width="1115" alt="Screenshot 2021-10-15 at 11 07 18" src="https://user-images.githubusercontent.com/537751/137453908-1b44b458-73a7-44b1-bd4b-6ec71b6674ad.png">

## On Sale Products
<img width="1490" alt="Screenshot 2021-10-15 at 11 08 35" src="https://user-images.githubusercontent.com/537751/137454073-10802057-4b2a-432e-a461-1992a6e20c76.png">

## Top Rated Products 
<img width="1042" alt="Screenshot 2021-10-15 at 11 09 22" src="https://user-images.githubusercontent.com/537751/137454256-04450f2d-d093-43f6-bc20-3e129df72bfc.png">

## Order Summary
<img width="514" alt="Screenshot 2021-10-15 at 11 10 52" src="https://user-images.githubusercontent.com/537751/137454419-e07818c3-662b-4c5a-a252-b30dca686aee.png">

## Cart & Cart i2
<img width="987" alt="Screenshot 2021-10-15 at 11 10 57" src="https://user-images.githubusercontent.com/537751/137454431-8e131df1-3a43-4123-8553-d0af50d87582.png">

## Checkout
<img width="935" alt="Screenshot 2021-10-15 at 11 11 05" src="https://user-images.githubusercontent.com/537751/137454455-71ad8516-34fe-4607-9914-77f4677f7468.png">


### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. merge this PR
2. create a product with no image
3. create a new post and add the WooCommerce blocks and include the product created at step 2. to see its placeholder image and border



### Performance Impact



- estimated no performance impact since these are mostly design changes

### Changelog

> Improve the product images placeholder display by adding a light gray border to it.
